go/teams/handler.go

```diff
@@ -576,14 +576,15 @@ func HandleTeamSeitan(ctx context.Context, g *libkb.GlobalContext, msg keybase1.
 	tx := CreateAddMemberTx(team)
 
 	for _, seitan := range msg.Seitans {
+		// TODO what if there are two seitans for same inviteid?
```

Comment:
Ok, it'll still be caught by the prechecker anyway right? That being said since this is all in one TX I think I did this correctly within the transaction (`alreadyUsed := alreadyUsedBeforeTransaction + tx.usedInviteCount[inviteID]`), but would be good to write a test.